src/util/time.cpp

```diff
@@ -49,10 +49,10 @@ bool ChronoSanityCheck()
 
     // Check that the above zero time is actually equal to the known unix timestamp.
     struct tm epoch;
-#ifdef _MSC_VER
-    if (gmtime_s(&epoch, &time_val) != 0) {
-#else
+#ifdef HAVE_GMTIME_R
     if (gmtime_r(&time_val, &epoch) == nullptr) {
+#else
+    if (gmtime_s(&epoch, &time_val) != 0) {
 #endif
```

Comment:
The upstream commit being backported was merged before `ChronoSanityCheck` was added, so why was this change not made upstream then?

Please add a `Zcash: Some description` message to the commit message that notes what the changes are relative to the upstream commit and why (so they are preserved in git history).